### DISCUSSION TITLE
第一屏故障文字过长时无法完全显示

### DISCUSSION THREAD

**Discussion by zxc7895531** [LIlGG/halo-theme-sakura#403] [General]
### 目前使用的 Sakura 版本？

2.0.0

### 目前使用的 halo 版本？

2.6.0

### 建议/问题

第一屏故障文字过长时无法完全显示，建议添加字体大小自适应或手动调节设置

**Comment by LIlGG** [comment]
故障文字不建议设置太多，相当于一个醒目的提示语。
如果你指的是故障文字下方的网站副标题，那么在下个 Beta.2 版本将会得到优化。例如

![image](https://github.com/LIlGG/halo-theme-sakura/assets/31335418/73d7d8ac-0b22-4ff1-8618-c0e6bcecc72f)
